src/diffusers/schedulers/scheduling_ddim_flax.py

```diff
@@ -125,7 +130,19 @@ def __init__(
         beta_schedule: str = "linear",
         set_alpha_to_one: bool = True,
         steps_offset: int = 0,
+        prediction_type: str = "epsilon",
+        **kwargs,
     ):
+        message = (
+            "Please make sure to instantiate your scheduler with `prediction_type` instead. E.g. `scheduler ="
+            " FlaxDDIMScheduler.from_pretrained(<model_id>, prediction_type='epsilon')`."
+        )
+        predict_epsilon = deprecate("predict_epsilon", "0.10.0", message, take_from=kwargs)
+        if predict_epsilon is not None:
+            prediction_type = "epsilon" if predict_epsilon else "sample"
+
+        self.prediction_type = prediction_type
```

Comment:
Here we need to modify the frozen dict as shown here: https://github.com/huggingface/diffusers/blob/9f10c545cbf54dd4d87e7e0f24e1ec02e928c966/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L115

Reply:
@patrickvonplaten I don't think there's an internal dict yet in the `__init__` function. I wrote this test to verify that it works as expected: https://github.com/huggingface/diffusers/pull/1393/commits/01df570cdf17501c80e524e7e6ae6fa33cf056b5#diff-bf88051e4d8be9ab9d5a5f24a0daac59c898ddb1fca8505728f49b3809cd3666R611-R620

Reply:
Ah yeah you're right good point! Fixing this on main as well